### ISSUE TITLE
Impossible to open speaker notes again after closing them

### ISSUE THREAD

**Issue by qwandor** [google/comprehensive-rust#182]
I can't find any way to open the speaker notes again after closing them once. Steps to reproduce:

1. Go to https://google.github.io/comprehensive-rust/.
2. Click the link to open speaker notes in a new window. The new window opens, and the link disappears from the main window.
3. Close the speaker notes window. I would expect the link in the main window to reappear at this point, but it does not.
4. Go to a different page in the main window. Still no link to reopen speaker notes.
5. Refresh the main window. There is still no link to reopen speaker notes.
6. Close the main window, and open https://google.github.io/comprehensive-rust/ again. There's still no link to reopen speaker notes again, they are gone forever.

**Comment by mgeisler** [comment]
Hey @qwandor,

Yes, the state is set to `popup` when you first open them in an external popup. You then get a close button in the top-right corner of the screen:

![image](https://user-images.githubusercontent.com/89623/213187353-7db04180-c287-4af2-876e-24c3025aa8b6.png)

Clicking this button will change the state back so that the notes appear inline again.

This is not ideal and I was afraid that it would be hard to notice the new button... I played with making it flash to red before fading back to grey. However, that was a bit too much CSS animation coding so I gave it up.

It used to be the case that closing the popup would make the notes appear inline again. However, this lead to another problem: if you paged quickly through the notes, you would end up in a broken state. I fixed that via #161.

In short, navigating to another page in the speaker looks the same to us as you closing the window. So it's hard to reliably react on you closing the window without also messing up the state when you navigate naturally between pages.

**Comment by mgeisler** [comment]
I'm slowly working towards a redesign of the course theme (#1089). There, the idea is to combine the "open popup" button and the "close popup" button into a single toggle button. That should make it much clearer where to click to bring the notes back into the main window.

**Comment by fw-immunant** [comment]
Seems to be fixed.
